### PR TITLE
FW-5385 Add search bar to app nav bar

### PR DESCRIPTION
**Pull request by bronson-c**
### Description of Changes
- Adds the same search bar from the site nav bar to the app nav bar
- App nav bar search is spaced the same as the app nav bar menu items

### Checklist

- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [x] ~~Tests have been updated / created if applicable~~

### Reviewers Should Do The Following:

- [X] Review the changes here
- [ ] Pull the branch and test locally

### Additional Notes

N/A

### CODE REVIEW

File: src/components/AppNavBar/AppNavBarPresentation.js [hunk 1]
```diff
@@ -166,6 +167,15 @@ function AppNavBarPresentation({ isHome = false, login, logout }) {
               {getIcon('QuestionCircleSolid', 'fill-current h-full w-6 mr-2')}
               Support
             </Link>
+            {/* Search */}
+            {!isHome && (
+              <div
+                id="AppNavSearch"
+                className="inline-flex items-center px-8 py-2"
```
Comment (bronson-c) [inline src/components/AppNavBar/AppNavBarPresentation.js:174]: The spacing of the search box looks fine when a user is logged in, however it looks a bit squished when the user is logged out and the icon is replaced by the "Sign in / Register" menu item. If this spacing needs to be adjusted please let me know

Comment (gmcauliffe) [reply src/components/AppNavBar/AppNavBarPresentation.js:174]: Try removing `md:w-1/2 2xl:w-1/4` from the ul classes on line 162. We're already justifying the links to the end so constricting the width isn't really necessary and should allow more space for the search.

**Review by gmcauliffe — APPROVED** [review]
Have a go at fixing the spacing when logged out if you can (see suggestion).
Please go ahead and merge when you're satisfied 👍

File: src/components/AppNavBar/AppNavBarPresentation.js (hunk 1 above)
Comment (bronson-c) [reply src/components/AppNavBar/AppNavBarPresentation.js:174]: Perfect, that fixed the spacing, thanks!

**Pull request merged**